aws_embedded_metrics/validator.py

```diff
@@ -57,14 +58,15 @@ def validate_dimension_set(dimension_set: Dict[str, str]) -> None:
             raise InvalidDimensionError("Dimension name cannot start with ':'")
 
 
-def validate_metric(name: str, value: float, unit: Optional[str]) -> None:
+def validate_metric(name: str, value: float, unit: Optional[str], storageResolution: Optional[int], metricNameAndResolutionMap: dict) -> None:  # noqa: E501
     """
     Validates a metric
 
         Parameters:
             name (str): The name of the metric
             value (float): The value of the metric
             unit (Optional[str]): The unit of the metric
+            storageResolution (Optional[int]): The storage resolution of metric
```

Comment:
Please updated the method signature parameter  for metricNameAndResolutionMap in method comments

Reply:
Done!